inputfiles/overridingTypes.json

```diff
@@ -235,23 +235,6 @@
                     }
                 }
             },
-            "HTMLLinkElement": {
```

Comment:
we have explicitly added these in https://github.com/Microsoft/TSJS-lib-generator/pull/488 and https://github.com/Microsoft/TSJS-lib-generator/pull/425 can we keep them for now.

Reply:
`relList` is in the spec so it's not really removed. `disable` is not, the spec does not mention it, even not as a deprecated attribute...

Reply:
Okay, it was once in the spec, [has been removed](https://github.com/whatwg/html/issues/1081) not because it's needless but it's in inappropriate document. I'll add it again.